### ISSUE TITLE
Serving the epub file from a servlet

### ISSUE THREAD

**Issue by phelgren** [futurepress/epub.js#539]
I can't quite figure out what the issue is here.  I got a demo running with a simple path to the file on a simple web server (Web server for Chrome) (inputfile.html example) where the js is var book = ePub("../reader/moby-dick.epub");  But I want to serve the files from a servlet and when I serve the file using an Outputstream I get an error because for some reason the URL to the servlet either gets mangled in some way or the epub.js seems to be making calls to the unzipped version or something.  I haven't walked through the js code yet (as you can tell) but before starting digging, is there some reason that this wouldn't work? e.g. retrieve the file and have ePub then open it? I saw a closed post (#425 I think) that talked about streams.  Is that the problem here or am I "just doing it wrong"?

Basically it seems like it treats the servlet method call as the path to the file, rather than a way to return the file.  The servlet URL is something like  http://www.myserver.com/EBookUtils/eBook?action=getBook&name=moby-dick.epub  and the error I see at line 4459 is a 404 on http://www.myserver.com/EBookUtilsMETA-INF/container.xml  which of course, is correct because nothing like that exists on the server.   I hope that is clear enough...

**Comment by ituaijagbone** [comment]
@phelgren Were you able to solve this issue? I am currently facing the same challenge.

**Comment by phelgren** [comment]
No.  I wasn't able to solve it.  I have been searching for alternatives and I will post here if I find one.
